compiler/passes/codegen.cpp

```diff
@@ -409,6 +409,61 @@ static void genFilenameTable() {
   genGlobalInt32(sizeName, gFilenameLookup.size());
 }
 
+//
+// This adds the Chapel symbol table to the config file
+// Our symbol table is formed by two 1-D arrays with 2 elements
+// per entry:
+//
+// chpl_funSymTable     = cname, Chapel name
+// chpl_filenumSymTable = Chapel file name index, Chapel line number
+//
+static void genUnwindSymbolTable(){
+  GenInfo *info = gGenInfo;
+
+  //TODO should handle LLVM
+  //TODO should be activated by a compiler flag
```

Comment:
Not sure it needs to be activated by a compiler flag. If you add CHPL_STACKTRACE or something along those lines, I'd expect that variable to have a version in driver.h; then this code could just check if( 0 == strcmp(CHPL_STACKTRACE, "none") return;